### PR TITLE
docs: adding Old Forge Double Crusted Stuffed White Pizza recipe to pizza-recipes.md

### PR DESCRIPTION
**Pull request by Emmarie-Ahtunan**
## Description

This PR adds Old Forge Style Double Crusted Stuffed White Pizza recipe to pizza-recipes.md

## What type of PR is this? (check all applicable)

- [X] 🍕 Feature
- [ ] 🐛 Bug Fix
- [X] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Relates to #32 

## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [X] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by Emmarie-Ahtunan** [comment]
I'm having a hard time adding the image file. I have the image and need to add it and link it in the .md file. I know I need to figure out how to do that before you can merge.

Thank you for the opportunity to learn to contribute to open source during my first Hacktoberfest!

**Comment by CBID2** [comment]
> I'm having a hard time adding the image file. I have the image and need to add it and link it in the .md file. I know I need to figure out how to do that before you can merge.
> 
> Thank you for the opportunity to learn to contribute to open source during my first Hacktoberfest!

Hi @Emmarie-Ahtunan! 😄 Welcome to OpenSauced! :) To add an image via Markdown, you do this:
`   Alt text `

**Comment by Emmarie-Ahtunan** [comment]
Thank you so much, @CBID2! I'm working on getting it added to the file first right now so that I can do that step. I think I found the answer to how to add it to the image file.

File: pizza-recipes.md
Comment (CBID2) [inline pizza-recipes.md:36]: [suggestion] While mentioning that you’re from NEPA is good, people might not know who is being referred to, so consider omitting that information.
```suggestion
Old Forge Double Crust White Pizza is something native to Northeast Pennsylvania.
 Also known as “Stuffed Pizza” to some, this pizza is essentially a giant (and delicious) grilled cheese sandwich!  It is flavorful to the extreme, great for gatherings and parties, and very versatile.  It also happens to be very easy to make from simple ingredients from the grocery store.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Emmarie-Ahtunan! :) I gave an in-line comment to improve your entry. Also, remove the `.DS_Store` file?

**Comment by CBID2** [comment]
Can you also fill out the PR form @Emmarie-Ahtunan? When you’re making a PR, it’s important to do this so maintainers have an idea about what you’re contributing.

**Comment by Emmarie-Ahtunan** [comment]
@CBID2 I think I covered everything. Thank you for your feedback, help, and any additional feedback and help! It means a lot to me!

**Comment by CBID2** [comment]
> @CBID2 I think I covered everything. Thank you for your feedback, help, and any additional feedback and help! It means a lot to me!

You're welcome @Emmarie-Ahtunan! 😄 One more thing, you need to link the issue that your PR solves in the `Related Issues & Documents` section in your PR form. It helps maintainers gain a understand the problem your PR solves. Here's how it's done:

`Relates to #issue`

In your PR, it would be `Relates to 32`

**Comment by Emmarie-Ahtunan** [comment]
> > @CBID2 I think I covered everything. Thank you for your feedback, help, and any additional feedback and help! It means a lot to me!
> 
> You're welcome @Emmarie-Ahtunan! 😄 One more thing, you need to link the issue that your PR solves in the `Related Issues & Documents` section of your PR form. It helps maintainers gain a understand the problem your PR solves. Here's how it's done:
> 
> `Relates to #issue`
> 
> In your PR, it would be `Relates to #32`

Thank you so very much, @CBID2! I've learned so much from you and @adiati98 today. I am so grateful! I can go fix the regional pizza now too with you additional feedback. TYSM! <3

**Review by CBID2 — APPROVED** [review]
Congratulations @Emmarie-Ahtunan!:) You just made your first contribution! 🥳 It was wonderful working with you! 😄 If you want to stay updated on OpenSauced, join our [Discord community](https://discord.com/invite/U2peSNf23P)! :) We'd love to have you! :)

**Comment by Emmarie-Ahtunan** [comment]
> Congratulations @Emmarie-Ahtunan!:) You just made your first contribution! 🥳 It was wonderful working with you! 😄 If you want to stay updated on OpenSauced, join our [Discord community](https://discord.com/invite/U2peSNf23P)! :) We'd love to have you! :)

I'm in the Discord and very excited to finally be engaging with OpenSauced. I learned about y'all from a former DevRel at GH! Again, thank you so much and looking forward to more contributions!

**Pull request merged**
